### DISCUSSION TITLE
Filesystem blocking on Windows OS

### DISCUSSION THREAD

**Discussion by thonglepas** [SafeExamBrowser/seb-win-refactoring#922] [Q&A]
Hi,

We are planning to create an assessment using SEB. The assessment will be run on candidate's BYO devices. During the assessment, the candidate will require to use some 3rd-party applications (Adobe apps, etc.) to complete the assessment.

Due to the ability to access filesystem via 3rd-party applications, is there any ways we can restrict the access for the filesystem except for some certain folders in SEB in Windows?

Thank you very much!

**Comment by dbuechel** [comment]
No, that is unfortunately not possible. Once you permit third-party applications, they will be available with their full feature set which often indeed includes a file system dialog for e.g. opening and saving data.

**Reply by dbuechel** [reply]
See also https://safeexambrowser.org/windows/win_usermanual_en.html#ApplicationsPane.
